recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg

```diff
@@ -0,0 +1,25 @@
+# =====================================================================
+# DomF guest configuration
+# =====================================================================
+
+seclabel='system_u:system_r:domU_t'
+
+# Guest name
+name = "DomF"
+
+# Kernel image to boot
+kernel = "/xt/domf/Image"
+
+# Kernel command line options
+extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domf ip=192.168.1.13 rw rootwait console=hvc0"
+
+# Initial memory allocation (MB)
+memory = 256
+
+# Number of VCPUS
```

Comment:
Why do you have tautological comments only for some  parameters? I prefer to remove them all

Reply:
I understand. But this does not mean, that we need to sustain this ill practice.

Reply:
ok

Reply:
done